lua/swenv/api.lua

```diff
@@ -185,6 +193,45 @@ M.pick_venv = function()
   end)
 end
 
+M.reset_venv = function()
+  -- Reset PATH to original value
+  vim.fn.setenv('PATH', ORIGINAL_PATH)
+
+  -- Reset virtual environment variables
+  if ORIGINAL_VIRTUAL_ENV ~= vim.NIL then
+    vim.fn.setenv('VIRTUAL_ENV', ORIGINAL_VIRTUAL_ENV)
+  else
+    vim.fn.setenv('VIRTUAL_ENV', nil)
+  end
+
+  -- Reset Conda environment variables
+  if ORIGINAL_CONDA_PREFIX ~= vim.NIL then
+    vim.fn.setenv('CONDA_PREFIX', ORIGINAL_CONDA_PREFIX)
+  else
+    vim.fn.setenv('CONDA_PREFIX', nil)
+  end
+
+  if ORIGINAL_CONDA_DEFAULT_ENV ~= vim.NIL then
+    vim.fn.setenv('CONDA_DEFAULT_ENV', ORIGINAL_CONDA_DEFAULT_ENV)
+    vim.fn.setenv('CONDA_PROMPT_MODIFIER', '(' .. ORIGINAL_CONDA_DEFAULT_ENV .. ')')
```

Comment:
I don't use conda but why should this be set like this? Is that normally changed when changing conda envs, should the original value also be captured?

Reply:
I used to use conda, but I've deprecated it, favoring `pyenv` with `poetry`. I can't be sure about this, would you like me to remove it?

Reply:
If there is no reason to set it then maybe yes. I was wondering since all the other variables were captured and restored but not this one.

Reply:
Should be removed now :)